01-Login/src/App.js

```diff
@@ -1,71 +1,100 @@
-import React, { Component } from 'react';
-import { Navbar, Button } from 'react-bootstrap';
-import './App.css';
+import React, { Component, Fragment } from "react";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import createAuth0Client from "@auth0/auth0-spa-js";
+import { Container } from "reactstrap";
+
+import PrivateRoute from "./components/PrivateRoute";
+import Loading from "./components/Loading";
+import NavBar from "./components/NavBar";
+import Footer from "./components/Footer";
+import Callback from "./views/Callback";
+import Home from "./views/Home";
+import Profile from "./views/Profile";
+
+// auth0 config
+import config from "./auth_config";
+
+// styles
+import "samples-bootstrap-theme/dist/css/auth0-theme.css";
```

Comment:
I still need to work out the details, but let's get this updated so that it pulls this dependency from a git submodule as opposed to a package. Let me work out how that will work and get back to you.